src/services/measure.service.js

```diff
@@ -279,11 +280,21 @@ const evaluateMeasure = async (args, { req }) => {
 const careGaps = async (args, { req }) => {
   logger.info('Measure >>> $care-gaps');
 
-  validateCareGapsParams(req.query);
+  let query;
+  if (req.method === 'POST') {
+    query = gatherParams(req.query, req.body);
```

Comment:
The `gatherParams()` function name isn't super descriptive but I don't necessarily think we should change the function name because otherwise it will be super long. Maybe we should add a comment here explaining that gatherParams creates a new object of the parameters from both the request query and request body